app/src/androidTest/java/ch/sdp/vibester/activity/CreateProfileActivityTest.kt

```diff
@@ -69,9 +86,9 @@ class CreateProfileActivityTest {
 
         val intent = Intent(ApplicationProvider.getApplicationContext(), CreateProfileActivity::class.java)
         intent.putExtra("email", mockEmail)
-        intent.putExtra("isUnitTest", true)
```

Comment:
Nooooo bye my friend isUnitTest, you had a great life 😂😂😂😂 !!!

Reply:
ahahahahahhahaa bye-bye and never come back please